csi/README.md

```diff
@@ -0,0 +1,15 @@
+# CVMFS CSI plugin
+
+CVMFS [Container Storage Interface](https://github.com/container-storage-interface/spec) (CSI) plugin provides read-only mounting of [CernVM-FS](https://cernvm.cern.ch/fs/) (CVMFS) repositories in CSI-enabled container orchestrators.
+
+See project documentation at [docs/](./docs).
+
+## Reporting bugs
+
+Please report issues at <https://github.com/cvmfs/csi/issues/new>.
```

Comment:
Link broken, either point to https://github.com/cvmfs-contrib/cvmfs-csi or this repo